src/tools/web/client/README.md

```diff
@@ -0,0 +1,17 @@
+# elektra-web client
+
+
+## Installation
+
+```
+npm install
```

Comment:
In which directory?

Tell where the dependencies are listed.

Installation fails with:
```
npm WARN package.json elektra-web-client@1.0.0 No description
npm WARN package.json elektra-web-client@1.0.0 No repository field.
npm WARN package.json webpack-dev-server@1.16.1 No README data
npm ERR! peerinvalid The package react does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer material-ui@0.15.4 wants react@^15.0.0
npm ERR! peerinvalid Peer react-dom@15.3.2 wants react@^15.3.2
npm ERR! peerinvalid Peer react-redux@4.4.5 wants react@^0.14.0 || ^15.0.0-0
npm ERR! peerinvalid Peer react-tap-event-plugin@1.0.0 wants react@^15.0.0-0
npm ERR! peerinvalid Peer react-treeview@0.4.5 wants react@>=0.14.0
npm ERR! peerinvalid Peer rgx@0.2.4 wants react@^0.14.0

npm ERR! System Linux 3.16.0-4-amd64
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! cwd /home/markus/Projekte/Elektra/libelektra/src/tools/web/client
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.4.21
npm ERR! code EPEERINVALID
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/markus/Projekte/Elektra/libelektra/src/tools/web/client/npm-debug.log
npm ERR! not ok code 0
```